src/main/java/org/candlepin/resource/EnvironmentResource.java

```diff
@@ -399,7 +399,9 @@ public AsyncJobStatusDTO promoteContent(@Verify(Environment.class) String envId,
         Set<String> contentIds;
         try {
             contentIds = this.batchCreate(contentToPromote, environment);
-            this.contentAccessManager.syncOwnerLastContentUpdate(environment.getOwner());
+            // TODO: Remove the line below once certificate generation based on the `lastContentUpdate` field
+            // in environments is implemented.
+            environment.getOwner().syncLastContentUpdate();
```

Comment:
Same as the other commit: these are extraneous.

Reply:
Removed